### DISCUSSION TITLE
how to switch from yi hack v4

### DISCUSSION THREAD

**Discussion by derDeno** [roleoroleo/yi-hack-Allwinner-v2#206] [Q&A]
Hi, i currently have two cams with the yi hack v4 installed. How can i now switch to the allwinner v2?
The normal installation procedure seems not to work.

booth are outdoor cams 1080p with base version 3.0.0.0D_201809111054, if this helps to resolve the issue.

thanks for any advice

**Comment by roleoroleo** [comment]
You can't switch from v4 to allwinner v2.
It's not the same cam.
